### DISCUSSION TITLE
Close Sprints and move the "not done issues" to another sprint

### DISCUSSION THREAD

**Discussion by Thom087** [Leantime/leantime#2069] [General]
I have several Sprints lets say one for October23 and one for November23. Now October Sprint is done and I would like to close that and move automatically all not done issue to the next Sprint November23. Is this possible like it is in Jira?

**Comment by gloriafolaron** [comment]
Thanks for asking! The system does not currently allow for this but it is something on our roadmap currently. We're still mapping out what this will look like in Leantime.

**Comment by Thom087** [comment]
Ok thanks a lot
